ultravox/data/datasets.py

```diff
@@ -41,11 +41,11 @@
     # from https://arxiv.org/pdf/2402.08846
     "Transcribe speech to text: <|audio|>",
     # from GPT-4
-    "Capture every word from <|audio|> verbatim",
-    "Convert speech to text from <|audio|>",
-    "Listen and transcribe the complete text from <|audio|>",
-    "Record in writing what is spoken in <|audio|>",
-    "Transcribe the spoken words from <|audio|> with exact wording and punctuation",
+    "Capture every word from the audio verbatim\n<|audio|>",
+    "Convert speech to text from audio\n<|audio|>",
+    "Listen and transcribe the complete text from audio\n<|audio|>",
+    "Record in writing what is spoken in audio\n<|audio|>",
+    "Transcribe the spoken words from audio with exact wording and punctuation\n<|audio|>",
```

Comment:
Yes, the model doesn't know, but we are using KD loss anyway, so it doesn't matter much. and I agree that using these prompts would get us good transcription results because the LLM doesn't know much about it.

I added a repeating prompt that seems to have high accuracy for repeating the text. I have it in another experiment, so that is not part of the v0.4 release.